packages/web-pkg/src/composables/pagination/usePagination.ts

```diff
@@ -6,6 +6,9 @@ import {
   useRouteQueryPersisted,
   PaginationConstants
 } from 'web-pkg/src/composables'
+import { eventBus } from 'web-pkg'
```

Comment:
To avoid weird round trips in our unit tests. This should make them work again.

```suggestion
import { eventBus } from 'web-pkg/src/services/eventBus'
```

Reply:
Done